experimental/regression_suite/shark-test-suite-models/sd3/test_clip.py

```diff
@@ -63,7 +63,7 @@
 )
 
 sd3_clip_mlir = fetch_source_fixture(
-    "https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sd3-prompt-encoder/model.mlirbc",
+    "https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sd3-prompt-encoder/model.mlir",
     group="sd3_clip",
```

Comment:
> @saienduri I regenerated the "outdated" mlirbc files and uploaded them as IR into the same name and directory but with text and as ".mlir" suffix. LMK what you think! :)

If continuing to use Azure for these files (remember, this is "experimental" code so the bar is currently low for organization and code quality), I'd rather we not use individual user names and do use dates or versions, so a path like
* `sharkpublic/sharktank_tests/models/sd3/prompt-encoder/model_2024_12_23.mlirbc`
* `sharkpublic/sharktank_tests/models/sd3/prompt-encoder/model_v3.1.0rc20241223.mlirbc`
* `sharkpublic/sharktank_tests/models/sd3-prompt-encoder-v3.1.0rc20241223.mlirbc`
* `sharkpublic/sharktank_tests/models/sd3/prompt-encoder/sd3-prompt-encoder-v3.1.0rc20241223.mlirbc`